R/tm_g_patient_profile.R

```diff
@@ -191,6 +191,7 @@ tm_g_patient_profile <- function(label = "Patient Profile Plot",
     ui_args = args,
     server = srv_g_patient_profile,
     server_args = list(
+      patient_id = patient_id,
```

Comment:
Additionally, the module also seems to use these boolean args:
```r
show_ae_plot = TRUE,
show_rs_plot = TRUE,
show_cm_plot = FALSE,
show_lb_plot = FALSE,
```

But, I think we can just remove them. It seems like a remnant of the past.

Reply:
yes. I have removed from upversioning app module.